libkbfs/md_ops_test.go

```diff
@@ -96,7 +96,13 @@ func verifyMDForPrivate(config *ConfigMock, rmds *RootMetadataSigned) {
 		gomock.Any(), TLFCryptKey{}).Return(&rmds.MD.data, nil)
 
 	config.mockKbpki.EXPECT().HasVerifyingKey(gomock.Any(), gomock.Any(),
-		gomock.Any(), gomock.Any()).AnyTimes().Return(nil)
+		gomock.Any(), gomock.Any(), gomock.Any()).AnyTimes().Return(nil)
+
+	if rmds.MD.IsFinal() {
+		config.mockCodec.EXPECT().Decode(gomock.Any(), gomock.Any()).Return(nil)
```

Comment:
These are from the MD deepCopy in VerifyRootMetadata?  Maybe you can add a comment explaining?